src/objects/scss/_fonts.scss

```diff
@@ -1,20 +0,0 @@
-@font-face {
```

Comment:
`docker compose up -d --build` fails with this error for me:
```
 => ERROR [web-init production 18/18] RUN python src/manage.py colle  1.5s
------
 > [web-init production 18/18] RUN python src/manage.py collectstatic --noinput:
1.180 Post-processing 'css/screen.css' failed!
1.180 
1.180 Traceback (most recent call last):
1.180   File "/app/src/manage.py", line 24, in <module>
1.180     execute_from_command_line(sys.argv)
1.180   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
1.180     utility.execute()
1.180   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 436, in execute
1.180     self.fetch_command(subcommand).run_from_argv(self.argv)
1.180   File "/usr/local/lib/python3.10/site-packages/django/core/management/base.py", line 412, in run_from_argv
1.180     self.execute(*args, **cmd_options)
1.180   File "/usr/local/lib/python3.10/site-packages/django/core/management/base.py", line 458, in execute
1.180     output = self.handle(*args, **options)
1.180   File "/usr/local/lib/python3.10/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 209, in handle
1.180     collected = self.collect()
1.180   File "/usr/local/lib/python3.10/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 154, in collect
1.180     raise processed
1.180   File "/usr/local/lib/python3.10/site-packages/django/contrib/staticfiles/storage.py", line 372, in _post_process
1.180     content = pattern.sub(converter, content)
1.180   File "/usr/local/lib/python3.10/site-packages/django/contrib/staticfiles/storage.py", line 249, in converter
1.180     hashed_url = self._url(
1.180   File "/usr/local/lib/python3.10/site-packages/django/contrib/staticfiles/storage.py", line 182, in _url
1.180     hashed_name = hashed_name_func(*args)
1.180   File "/usr/local/lib/python3.10/site-packages/django/contrib/staticfiles/storage.py", line 422, in _stored_name
1.180     cache_name = self.clean_name(self.hashed_name(name))
1.180   File "/usr/local/lib/python3.10/site-packages/django/contrib/staticfiles/storage.py", line 143, in hashed_name
1.180     raise ValueError(
1.180 ValueError: The file 'admin/fonts/Roboto-Bold-webfont.woff' could not be found with <django.contrib.staticfiles.storage.ManifestStaticFilesStorage object at 0x7f3199548280>.
------
failed to solve: process "/bin/sh -c python src/manage.py collectstatic --noinput" did not complete successfully: exit code: 1
```

Does it work for you?

Reply:
Now it works, you are right!